Software-Engineering/L5-Principal-Software-Engineer.md

```diff
@@ -10,27 +10,33 @@
 ## 🦉 Domain expertise
 
 - I contribute to core technologies used by a variety of teams.
-- I help make critical product architecture and implementation decisions.
-- I maintain a vision of the overall technical direction of my product lines.
-- I help define and build out technical standards and principles for how software is built throughout the company.
-- I manage company-wide risks, and architect to reduce risk.
+- I help teams make critical product architecture and implementation decisions.
+- I maintain a vision of the overall technical direction within my impact radius.
+- I contribute to the engineering standards and principles of the company.
+- I contribute to the Octopus technical strategy.
+- I manage technical risks, and architect to reduce technical risk.
+- I collaborate with other Principal Software Engineers to ensure we are making cohesive and complimentary decisions that align with the Octopus technical strategy.
 
 <details>
 <summary>Examples</summary>
 
 - I successfully completed complex tasks spanning multiple domains and teams with high impact. 
-- I helped a team reduce complexity through sound architectural thinking.
-- I identified a problem that was timely, important, and impactful, and worked closely with product, engineering, and executive stakeholders to articulate and refine a solution.
-- I handled multiple conflicting priorities and sequenced work resulting in the best collective outcome.
+- I helped a team reduce complexity and risk through sound architectural thinking.
+- I developed a plan to evolve the architecture of one of the core components of Octopus within my impact radius.
+- I identified a problem that was timely, important, and impactful, and worked closely with product, engineering, and executive stakeholders to articulate and refine a solution and delivery plan.
+- I worked with teams to manage multiple conflicting priorities and helped sequence work resulting in the best collective outcome.
+- I collaborated with another principal engineer to ensure that a feature I was helping a team with would support a feature I knew another team was planning.
```

Comment:
🤔 I'm not sure we can dictate the scope of projects. Would it be enough for us to connect the relevant teams so they can make the best decision?

Reply:
I don't think we can dictate it, but I think we can influence it from time to time, and that influence can be important and timely.